src/seedu/addressbook/data/person/Person.java

```diff
@@ -66,6 +66,19 @@ public void setTags(UniqueTagList replacement) {
         tags.setTags(replacement);
     }
 
+    @Override
```

Comment:
You do really need two getPrintableString() methods here. Also as a utility method, the getPrintableString(Printable... printables) is better to be made a static method. You can refer to the code of your peers if you are not sure. Ideally, this method should also be used somewhere in the code base. Meanwhile, can you think of a better place to put this method. Think of its functionality. Hint: formatting.